integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdownMS.java

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
```

Comment:
Did that file exist in 2019? If so, you have to leave that in there

Reply:
it is a new file, I have split original ItPodShutdown.java  into 3 files : Base, ShutdownMs and ShutdownOptionsDomainEnvCluster.

Reply:
if you copied most of the old file over, i would leave the 2019 in there please